cloudbuild.yaml

```diff
@@ -35,6 +35,8 @@ steps:
       - run
       - "--rm"
       - snuba-test
+    env:
+      - 'SNUBA_MODE=$SNUBA_MODE'
```

Comment:
Should this be `SNUBA_MODE=ci`? I don't think there is any variable SNUBA_MODE defined here

Reply:
trying it rn!